p2p/sender.go

```diff
@@ -147,24 +149,54 @@ func withRelayRetry(fn func() error) error {
 	return err
 }
 
+type sendRecvOpts struct {
```

Comment:
add godoc

Reply:
I'm not such a big of comment like `sendRecvOpts defines SendReceive options` since the signal to noise isn't great. And this is unexported, so not sure it is worth it.